README.md

```diff
@@ -37,43 +38,49 @@ git clone git@github.com:AvaProtocol/ap-operator-setup.git
 cd ap-operator-setup
 ```
 
-We had two directory call `holesky` and `mainnet`. To setup testnet, you will do
+We had two directory call `holesky` and `ethereum`. To setup testnet, you will do
```

Comment:
I suggest us to use the actual name of the chain `ethereum` other than `mainnet`, because in future we might deploy to other "mainnets"